.travis.yml

```diff
@@ -19,7 +19,7 @@ env:
 
 matrix:
   allow_failures:
-    - php: 7.0
+    - php: hhvm
```

Comment:
i have no worries making the tests skip things around streams + hhvm. but i would like to keep the hhvm build green. at liip we do use it in some projects, for example. skipping tests should not be a big hassle. if we end up with most things not working on hhvm, we can reconsider.